modules/diskq/qdisk.c

```diff
@@ -494,7 +493,7 @@ _load_queue(QDisk *self, GQueue *q, gint64 q_ofs, guint32 q_len, guint32 q_count
           return FALSE;
         }
       sa = serialize_string_archive_new(serialized);
-      for (i = 0; i < q_count; i++)
+      for (gsize i = 0; i < q_count; i++)
```

Comment:
Why not `guint32`?

Reply:
I wanted to be "future" proof by having a sufficiently large storage. But you ar  rignt, technically the type should be the same of q_count. I would like to leave this as it is.

Reply:
Sure, thanks for the explanation. :)